src/test/java/com/znsio/common/JsonPlaceholder.feature

```diff
@@ -0,0 +1,21 @@
+Feature: JsonPlaceholder Utilities
+
+  @jsonPlaceholderUtilities @template
+  Scenario: JsonPlaceholder common functions
+    * def buildQuery =
```

Comment:
This is the worst case , in realtime scenario, you might not need such function. Try implementing things in simpler way.

Reply:
Removed above method and added 'query' column in Examples section.